---
audioLength: '20470116'
audioType: audio/mpeg
audioUrl: https://feeds.soundcloud.com/stream/679961927-microbinfie-what-software-not-to-write-1.mp3
author: Microbial Bioinformatics
date: '2019-09-19T00:00:00.000Z'
duration: 00:23:52
explicit: 'no'
guests: []
guid: tag:soundcloud,2010:tracks/679961927
image: https://i1.sndcdn.com/artworks-hDzuZa9pRT5j9Lu2-hW4fUw-t3000x3000.png
link: https://soundcloud.com/microbinfie/what-software-not-to-write-1
subtitle: In this episode we identify areas of “Peak-bioinf…
tags:
- microbinfie
- podcast
title: 'Episode 01: What bioinformatics software not to write part 1'
transcript: /microbinfie-transcripts/episode-01.txt
---

In this episode, we identify areas of "Peak Bioinformatics." There's a vast array of existing bioinformatics software—often, the tool you want to create already exists, or a new tool cannot offer significant improvements. We explore this topic in the contexts of genome assembly, read mapping, and phylogenetics.

### When Novel Software May Not Be Needed:

- **Plethora of Existing Tools:** There are already many tools available in the field.
- **Problem Resolution Status:** The problem may be more or less solved, or it has been proven unsolvable.
- **Obsolete Technology or Problems:** The underlying technology or problem might be outdated or have been surpassed by other methods.

### Multiple Sequence Aligners:
- **MAFFT:** [Website](https://mafft.cbrc.jp/alignment/software/)
- **MUSCLE:** [Website](https://www.drive5.com/muscle/)

### Whole Genome Aligners:
- **Mauve:** [Website](http://darlinglab.org/mauve/mauve.html)
- **Mugsy:** [Website](http://mugsy.sourceforge.net/)
- **Sibelia:** [Website](http://bioinf.spbau.ru/sibelia)
- **Parsnp:** [GitHub](https://github.com/marbl/parsnp)

### Assemblers:
- **SPAdes:** [GitHub](https://github.com/ablab/spades)
- **Skesa:** [GitHub](https://github.com/ncbi/SKESA)
- **Velvet:** [Website](https://www.ebi.ac.uk/~zerbino/velvet/)
- **Abyss:** [GitHub](https://github.com/bcgsc/abyss)
- **Edena:** [Website](http://www.genomic.ch/edena.php)
- **Ray:** [SourceForge](http://denovoassembler.sourceforge.net/)

### Long Read Assemblies:
- **HGAP:** [Source](https://github.com/PacificBiosciences/Bioinformatics-Training/wiki/HGAP-2.0)
- **Flye:** [GitHub](https://github.com/fenderglass/Flye)
- **Canu:** [GitHub](https://github.com/marbl/canu)
- **Ra:** [bioRxiv](https://www.biorxiv.org/content/10.1101/656306v1)
- **Unicycler:** [GitHub](https://github.com/rrwick/Unicycler)

### Read Mapping:
- **BWA:** [Website](http://bio-bwa.sourceforge.net/)
- **Bowtie2:** [Website](http://bio-bwa.sourceforge.net/)
- **Minimap2:** [GitHub](https://github.com/lh3/minimap2)  
  *(Note: BWA is better for short reads. [Blog post](https://lh3.github.io/2018/04/02/minimap2-and-the-future-of-bwa))*
- **BBtools:** [JGI](https://jgi.doe.gov/data-and-tools/bbtools/)
- **BLAST/BLAT:** [Website](https://genome.ucsc.edu/FAQ/FAQblat.html)
- **SMALT:** [Sanger](https://www.sanger.ac.uk/science/tools/smalt-0)
- **Snippy:** [GitHub](https://github.com/tseemann/snippy)
- **Phenix:** [GitHub](https://github.com/phe-bioinformatics/PHEnix)

### Variant Callers:
- **GATK:** [Website](https://software.broadinstitute.org/gatk/)
- **VIPR:** [ViPR](https://www.viprbrc.org/brc/home.spg?decorator=vipr)
- **Varscan2:** [SourceForge](http://varscan.sourceforge.net/)

### Workflow Managers:
- **Bespoke Example:** [GitHub](https://github.com/VertebrateResequencing/vr-codebase)
- **Snakemake:** [Documentation](https://snakemake.readthedocs.io/en/stable/)
- **Nextflow:** [Website](https://www.nextflow.io/)
- **Galaxy:** [Galaxy](https://usegalaxy.org/)
- **Bpipe:** [GitHub](https://github.com/ssadedin/bpipe)

### Phylogenetics:
- **RAXML - RAXML-NG:** [Software](https://cme.h-its.org/exelixis/software.html)
- **IQTREE:** [Website](http://www.iqtree.org/)
- **FastTree:** [Microbes Online](http://www.microbesonline.org/fasttree/)
- **BEAST 1 & 2:** [Website](https://www.beast2.org/)
- **RevBayes:** [Website](https://revbayes.github.io/)

### Metagenomics:
- **Taxonomic Classification Tools:** *Megan, Kraken, SIGMA, MIDAS, MetaPhlAn2, mOTUs.*
- **Assemblers:** *MetaSPAdes, MetaFlye, MEGAHIT, MetaVelvet.*  
  (*Many single isolate assemblers have been adapted for metagenomes.*  
   [Kalamari GitHub](https://github.com/lskatz/Kalamari))

### Antimicrobial Resistance (AMR):
- **Resources:** 
  - [AMR Toolkit GitHub](https://github.com/arpcard/amr_curation) 
  - [AMR Protocols](https://food-safety-bioinformatics-hackathon.github.io/AMR-protocols/)
- **ABRicate:** [GitHub](https://github.com/tseemann/abricate)
- **ARIBA:** [Sanger](https://www.sanger.ac.uk/science/tools/ariba)
- **Detection Tools:** 
  - [Detection Tools Spreadsheet](https://docs.google.com/spreadsheets/d/18XGWpDiaE249qQKDAL7gdBCka0Z1drpA_s3FElfMJe0/edit#gid=0)

### Other Mentioned Resources:
- Recent Review: *Zhang W, Chen J, Yang Y, Tang Y, Shang J, Shen B (2011). "A Practical Comparison of De Novo Genome Assembly Software Tools for Next-Generation Sequencing Technologies."* PLoS ONE, 6(3), e17915. [DOI](https://doi.org/10.1371/journal.pone.0017915)
- **Assemblerthon:** [Website](https://assemblathon.org/)
- **Blog Post:** *"Minimap2 and the Future of BWA:"* [Blog Post](https://lh3.github.io/2018/04/02/minimap2-and-the-future-of-bwa)
- **The Science Web:** [WordPress](https://thescienceweb.wordpress.com/2015/03/23/each-bioinformatician-to-have-their-own-personal-short-read-aligner-by-2016/)

[Episode 01 transcript](/microbinfie-transcripts/episode-01.txt)