---
date: 2019-09-19 00:00:00
title: MicroBinfie Podcast, 01 What bioinformatics software not to write part 1
link: https://soundcloud.com/microbinfie/what-software-not-to-write-1
tags:
  - microbinfie
  - podcast
---
# Episode Summary: Peak Bioinformatics

In this episode, we delve into the concept of "Peak Bioinformatics," exploring areas where further development of bioinformatics software may be redundant or ineffective. With numerous existing tools, we question if new software can provide significant improvements, focusing on discussions around genome assembly, read mapping, and phylogenetics.

For questions and comments, reach us at: [microbinfie@gmail.com](mailto:microbinfie@gmail.com)

## Show Notes

### When Novel Software is Unnecessary
- **Saturation of Tools**: Many tools already exist.
- **Problems Solved or Unsolvable**: Some challenges have been addressed or proven unsolvable.
- **Obsolete Technology**: Outdated methods replaced by newer techniques.

### Multiple Sequence Aligners
- [MAFFT](https://mafft.cbrc.jp/alignment/software/)
- [MUSCLE](https://www.drive5.com/muscle/)

### Whole Genome Aligners
- [Mauve](http://darlinglab.org/mauve/mauve.html)
- [Mugsy](http://mugsy.sourceforge.net/)
- [Sibellia](http://bioinf.spbau.ru/sibelia)
- [Parsnp](https://github.com/marbl/parsnp)

### Assemblers
- Short Reads:
  - [SPADES](https://github.com/ablab/spades)
  - [Skesa](https://github.com/ncbi/SKESA)
  - [Velvet](https://www.ebi.ac.uk/~zerbino/velvet/)
  - [Abyss](https://github.com/bcgsc/abyss)
  - [Edena](http://www.genomic.ch/edena.php)
  - [Ray](http://denovoassembler.sourceforge.net/)
- Long Reads:
  - [HGAP](https://github.com/PacificBiosciences/Bioinformatics-Training/wiki/HGAP-2.0)
  - [Flye](https://github.com/fenderglass/Flye)
  - [Canu](https://github.com/marbl/canu)
  - [Ra](https://www.biorxiv.org/content/10.1101/656306v1)
  - [Unicycler](https://github.com/rrwick/Unicycler)

### Read Mapping
- [BWA](http://bio-bwa.sourceforge.net/)
- [Bowtie2](http://bio-bwa.sourceforge.net/)
- [Minimap2](https://github.com/lh3/minimap2) (Note: [BWA recommended for short reads](https://lh3.github.io/2018/04/02/minimap2-and-the-future-of-bwa))
- [BBtools](https://jgi.doe.gov/data-and-tools/bbtools/)
- [BLAST/BLAT](https://genome.ucsc.edu/FAQ/FAQblat.html)
- [SMALT](https://www.sanger.ac.uk/science/tools/smalt-0)
- [Snippy](https://github.com/tseemann/snippy)
- [Phenix](https://github.com/phe-bioinformatics/PHEnix)

### Variant Callers
- [GATK](https://software.broadinstitute.org/gatk/)
- [VIPR](https://www.viprbrc.org/brc/home.spg?decorator=vipr)
- [Varscan2](http://varscan.sourceforge.net/)

### Workflow Managers
- [Bespoke example](https://github.com/VertebrateResequencing/vr-codebase)
- [Snakemake](https://snakemake.readthedocs.io/en/stable/)
- [Nextflow](https://www.nextflow.io/)
- [Galaxy](https://usegalaxy.org/)
- [Bpipe](https://github.com/ssadedin/bpipe)

### Phylogenetics
- [RAxML - RAxML-NG](https://cme.h-its.org/exelixis/software.html)
- [IQTREE](http://www.iqtree.org/)
- [FastTree](http://www.microbesonline.org/fasttree/)
- [BEAST 1 & 2](https://www.beast2.org/)
- [RevBayes](https://revbayes.github.io/)

### Metagenomics
- **Taxonomic Classification**: Megan, Kraken, SIGMA, MIDAS, metaphlan2, mOTUs.
- **Assemblers**: MetaSpades, metaflye, MEGAHIT, MetaVelvet.
- [AMR Protocols](https://food-safety-bioinformatics-hackathon.github.io/AMR-protocols/)
- [ABRICATE](https://github.com/tseemann/abricate)
- [ARIBA](https://www.sanger.ac.uk/science/tools/ariba)

### Resources
- [Recent Review: Zhang W, et al. (2011) "A Practical Comparison of De Novo Genome Assembly Software Tools"](https://doi.org/10.1371/journal.pone.0017915)
- [The Assemblerthon](https://assemblathon.org/)
- Blog post: [BWA and Short Reads](https://lh3.github.io/2018/04/02/minimap2-and-the-future-of-bwa)
- [The Science Web Blog Post](https://thescienceweb.wordpress.com/2015/03/23/each-bioinformatician-to-have-their-own-personal-short-read-aligner-by-2016/)

Feel free to explore these resources and contribute to the ongoing discussion on the necessity and evolution of bioinformatics software.

### Extra notes

- **Software Saturation**: The conversation emphasizes the redundancy in developing new bioinformatics tools since many problems are already well-solved or obsolete. Creating new tools often seems unnecessary unless there is a truly novel approach.

- **Microarray vs. RNA-Seq**: Despite being largely replaced by RNA-Seq, microarray usage continued to be prevalent in literature, highlighting a lag in technology adoption.

- **Sequence Alignment**: Established tools like MAFT, MUSCLE, and BLAST are foundational for multiple sequence alignment, with newer methods offering only marginal improvements. Whole-genome alignment requires different tools like MOVE and MUG-C.

- **Genome Assemblers**: Popular tools include Spades, Velvet, and Skiza. Long-read assembly options like HGaP and Canoe are preferred for specific applications, but the quality of the data largely influences assembly outcomes. Smarts in assembly tool selection are favored over developing new assemblers from scratch.

- **Read Mapping**: The problem of short-read mapping is considered solved with tools like BWA and Bowtie. Minimap2 is suggested for long reads, though BWA is still favored for short reads.

- **Variant Calling**: There's a wide array of tools available. Optimizing parameter settings often outstrips the need to develop new variant calling software due to the existing options' comprehensiveness.

- **Bioinformatics Workflows**: The importance of workflow managers like SnakeMake, Nextflow, and Galaxy is highlighted for managing analyses efficiently, offering power and flexibility through container integration.

- **Phylogenetics**: The field is saturated with robust tools such as RAxML, IQ-TREE, and BEAST. New entrants face significant challenges in competing with such well-established tools.

- **Technical Constraints**: Development with FPGA for speeding up alignment tasks is intriguing but requires significant technical expertise beyond most individual bioinformaticians.

- **Community and Collaboration**: The content and opinions shared are part of a broader community effort and reflect personal views, not official institutional positions, emphasizing the collaborative nature of bioinformatics work.

