---
audioLength: '21074057'
audioType: audio/mpeg
audioUrl: https://feeds.soundcloud.com/stream/1898535225-microbinfie-128-haiti-cholera-outbreak-with-christine-lee-and-cynney-walters.mp3
author: Microbial Bioinformatics
date: '2024-09-19T05:00:05.000Z'
duration: 00:21:57
explicit: 'no'
guests: []
guid: tag:soundcloud,2010:tracks/1898535225
image: https://i1.sndcdn.com/artworks-I2y1XvutpfnPKpsD-wuz3NA-t3000x3000.png
link: https://soundcloud.com/microbinfie/128-haiti-cholera-outbreak-with-christine-lee-and-cynney-walters
subtitle: Andrew and Lee talk with Christine and Cynney abo…
tags:
- microbinfie
- podcast
title: 'Episode 128: Haiti cholera outbreak with Christine Lee and Cynney Walters'
transcript: /microbinfie-transcripts/episode-128.txt
---

- *Vibrio cholerae* has reemerged in Haiti in 2022.
- The outbreak reveals genetic similarities to strains that previously circulated in the region.
- We discussed Walters et al. ["Genome sequences from a reemergence of *Vibrio cholerae* in Haiti, 2022 reveal relatedness to previously circulating strains"](https://journals.asm.org/doi/abs/10.1128/jcm.00142-23)

The discussion centered around the implications of the genome sequences study, highlighting the evolutionary dynamics of the strains and the public health strategies needed to control and prevent cholera outbreaks in Haiti.

## Guests

- Christine Lee 
- [Cynney Walters](https://www.linkedin.com/in/cynney-walters-763111190)

### Extra notes

- **Whole Genome Sequencing (WGS):** Key to tracking and characterizing epidemiological trends of pathogens like *Listeria*, *Yersinia*, *Vibrio*, and other *Enterobacteriaceae*. It was notably used to analyze an outbreak in Haiti.

- **Haiti Cholera Outbreak:** In 2010, a deadly strain of *Vibrio cholerae* emerged after environmental devastation. Whole genome sequencing was used to study source attribution and found clustering between Nepalese and Haitian genomes.

- **Phylogenetic Analysis:** Utilized to trace the lineage of cholera in Haiti, suggesting a connection to the 2010 outbreak. Analysis involved comparing single nucleotide polymorphisms (SNPs) to understand strain relations over time.

- **Lifesat Tool:** Used to identify genetic variants, providing insights into cholera strain evolution. It masks potential mobile elements or phage-like sequences to refine evolutionary models.

- **Challenges of Environmental and Clinical Surveillance:** Lack of consistent environmental sampling leads to gaps in understanding pathogen resurgence, as seen with cholera’s re-emergence in Haiti. Surveillance is hindered by resource constraints and geopolitical factors.

- **Technical Challenges:** Special mention of the superintegron on *Vibrio cholerae’s* second chromosome complicating genomic analyses due to its dynamic nature in genetic sequences.

- **Antimicrobial Resistance (AMR):** Monitoring AMR profiles using tools like ResFinder is critical. The study found a consistent AMR profile among the majority of strains, with some variations detected.

- **Bioinformatics Contribution**: Emphasized the importance of bioinformatics in understanding the persistence and evolution of pathogens, and in tackling challenges like metagenomic analyses to differentiate between environmental and infectious strains.

- **Surveillance and Data Needs:** Advocates for routine, thorough genomic surveillance to bridge the gap in current outbreak data and historical trends. Environmental factors and human behaviors significantly impact microbial traceability.

- **Global Health Implications:** Highlights the impact of water infrastructure and climatic changes on cholera persistence. Calls for improved global surveillance as human movement and environmental shifts influence disease patterns.

[Episode 128 transcript](/microbinfie-transcripts/episode-128.txt)