---
audioLength: '30355200'
audioType: audio/mpeg
audioUrl: https://feeds.soundcloud.com/stream/967805281-microbinfie-41-sars-cov-2-phylogenomics-questions-answered.mp3
author: Microbial Bioinformatics
date: '2021-01-19T02:59:59.000Z'
duration: 00:25:17
explicit: 'no'
guests: []
guid: tag:soundcloud,2010:tracks/967805281
image: https://i1.sndcdn.com/artworks-hDzuZa9pRT5j9Lu2-hW4fUw-t3000x3000.png
link: https://soundcloud.com/microbinfie/41-sars-cov-2-phylogenomics-questions-answered
subtitle: ARTICnetwork & CLIMB-BIG-DATA present a panel disâ€¦
tags:
- microbinfie
- podcast
title: 'Episode 41: SARS-CoV-2 Phylogenomics questions answered'
transcript: /microbinfie-transcripts/episode-41.txt
---

# Panel Discussion on SARS-CoV-2 Phylogenomics

**Presented by:** ARTICnetwork & CLIMB-BIG-DATA

**Panelists:**

- **Nick Loman** - University of Birmingham
- **Verity Hill** - University of Edinburgh
- **Andrew Page** - Quadram Institute
- **Anna Price** - MRC CLIMB and Cardiff University

This discussion was part of a broader workshop focused on COVID-19 data analysis.

## Topics Covered:

- **More about Polecat**: Understanding its utility in the analysis of SARS-CoV-2.
- **COG-UK Phylotypes vs. Pangolin Lineages**: Exploring the differences between these classifications.
- **Civet and Llama**: Differentiating these tools and their use case scenarios.
- **Using BLAST for SARS-CoV-2**: Can it identify similar genomes?
- **Finding Similar Sequences**: Techniques for placing sample sequences in context using public repositories.
- **Manual Curation for PangoLEARN**: Understanding the process and its importance.
- **Subtree Selection in Civet**: Criteria and methods used in defining subtrees.
- **Adaptation of Civet for Other Viruses**: Exploring broader applications beyond SARS-CoV-2.
- **Vaccine Sequences Availability**: Where and how these sequences can be accessed.
- **Databases Reporting Variation**: An overview of resources tracking genomic variations.
- **Quality of Variant Calls**: Discussing the reliability and accuracy of variant assessments.

## Groups and Resources:

- [ARTIC and CLIMB-BIG-DATA Joint Workshop](https://www.climb.ac.uk/artic-and-climb-big-data-joint-workshop/)
- [CLIMB](https://www.climb.ac.uk/)
- [ARTIC Network](https://artic.network/)
- [COG-UK Consortium](https://cogconsortium.uk/)

## Software Tools:

- [Polecat](https://github.com/COG-UK/polecat)
- [Pangolin](https://github.com/cov-lineages/pangolin)
- [Llama](https://github.com/cov-lineages/llama)
- [Civet](https://github.com/artic-network/civet)
- [PangoLEARN](https://github.com/cov-lineages/pangoLEARN)
- [FigTree](http://tree.bio.ed.ac.uk/software/figtree/)

## Analysis Websites:

- [Cov-Lineages](https://cov-lineages.org/)
- [Nextstrain Clades](https://clades.nextstrain.org/)
- [Pangolin COG-UK](https://pangolin.cog-uk.io/)
- [Cov-Glue](http://cov-glue.cvr.gla.ac.uk/)

This panel served as an informative session shedding light on the nuances of SARS-CoV-2 phylogenomics, including methodological insights and tools for genome analysis.

[Episode 41 transcript](/microbinfie-transcripts/episode-41.txt)