---
date: 2021-01-19 00:00:00
title: MicroBinfie Podcast, 41 SARS-CoV-2 Phylogenomics questions answered
link: https://soundcloud.com/microbinfie/41-sars-cov-2-phylogenomics-questions-answered
tags:
  - microbinfie
  - podcast
---
ARTICnetwork & CLIMB-BIG-DATA present a panel discussion on SARS-CoV-2
phylogenomics with Nick Loman from the University of Birmingham,
Verity Hill from the University of Edinburgh, Andrew Page from the
Quadram Institute and Anna Price from MRC CLIMB and Cardiff
University. This was part of a workshop on COVID-19 data analysis.
The topics covered are: More about Polecat  Whats the difference
between COG-UK Phylotypes and Pangolin lineages? What is the
difference between Civet and Llama Can you use BLAST to find similar
SARS-CoV-2 genomes? How do you find similar sequences in the public
repositories to give your samples context? How do you manually curate
the dataset for PangoLEARN? How does Civet choose what constitutes a
subtree? Can Civet be adapted to other viruses? Are the vaccine
sequences available? Databases that report variation Quality of
variant calls?  Groups: https://www.climb.ac.uk/artic-and-climb-big-
data-joint-workshop/ https://www.climb.ac.uk/ https://artic.network/
https://cogconsortium.uk/  Software: https://github.com/COG-UK/polecat
https://github.com/cov-lineages/pangolin https://github.com/cov-
lineages/llama https://github.com/artic-network/civet
https://github.com/cov-lineages/pangoLEARN
http://tree.bio.ed.ac.uk/software/figtree/  Analysis websites:
https://cov-lineages.org/ https://clades.nextstrain.org/
https://pangolin.cog-uk.io/ http://cov-glue.cvr.gla.ac.uk/

