---
date: 2021-01-19 00:00:00
title: MicroBinfie Podcast, 41 SARS-CoV-2 Phylogenomics questions answered
link: https://soundcloud.com/microbinfie/41-sars-cov-2-phylogenomics-questions-answered
tags:
  - microbinfie
  - podcast
---
# Panel Discussion on SARS-CoV-2 Phylogenomics

## Presented by ARTIC Network & CLIMB-BIG-DATA

### Panelists:
- **Nick Loman**, University of Birmingham
- **Verity Hill**, University of Edinburgh
- **Andrew Page**, Quadram Institute
- **Anna Price**, MRC CLIMB and Cardiff University

### Workshop Topic:
This panel discussion was part of a workshop focused on COVID-19 data analysis. The panelists discussed various topics related to SARS-CoV-2 phylogenomics, including:

- Understanding Polecat
- Differences between COG-UK Phylotypes and Pangolin lineages
- Distinctions between Civet and Llama
- Using BLAST to identify similar SARS-CoV-2 genomes
- Finding similar sequences in public repositories to provide context for samples
- Manual curation of datasets for PangoLEARN
- Criteria used by Civet to define subtrees
- Adapting Civet for other viruses
- Availability of vaccine sequences
- Databases reporting variations
- Quality of variant calls

### Useful Links:

#### Workshop Information:
- [ARTIC and CLIMB-BIG-DATA Joint Workshop](https://www.climb.ac.uk/artic-and-climb-big-data-joint-workshop/)
- [CLIMB-BIG-DATA](https://www.climb.ac.uk/)
- [ARTIC Network](https://artic.network/)
- [COG-UK](https://cogconsortium.uk/)

#### Software Repositories:
- [Polecat](https://github.com/COG-UK/polecat)
- [Pangolin](https://github.com/cov-lineages/pangolin)
- [Llama](https://github.com/cov-lineages/llama)
- [Civet](https://github.com/artic-network/civet)
- [PangoLEARN](https://github.com/cov-lineages/pangoLEARN)
- [Figtree](http://tree.bio.ed.ac.uk/software/figtree/)

#### Analysis Websites:
- [Cov-Lineages](https://cov-lineages.org/)
- [Nextstrain Clades](https://clades.nextstrain.org/)
- [Pangolin Tool](https://pangolin.cog-uk.io/)
- [Cov-Glue](http://cov-glue.cvr.gla.ac.uk/) 

This discussion addressed key questions and provided insights into the ongoing efforts to understand the phylogenomics of SARS-CoV-2, contributing to global research and public health efforts against COVID-19.

