---
audioLength: '55802611'
audioType: audio/mpeg
audioUrl: https://feeds.soundcloud.com/stream/1019197834-microbinfie-55-sars-cov-2-and-sequencing-spike-with-sanger-sequencing-welcome-to-1996.mp3
author: Microbial Bioinformatics
date: '2021-04-08T04:00:01.000Z'
duration: 00:29:03
explicit: 'no'
guests:
- affiliation: Researcher in Computational Biology, Technical University of Denmark
  name: Kai Blin
  url: https://github.com/kblin
- affiliation: Researcher in Microbial Genomics, Technical University of Denmark
  name: Tue Sparholt Jørgensen 
  url: null
guid: tag:soundcloud,2010:tracks/1019197834
image: https://i1.sndcdn.com/artworks-hDzuZa9pRT5j9Lu2-hW4fUw-t3000x3000.png
link: https://soundcloud.com/microbinfie/55-sars-cov-2-and-sequencing-spike-with-sanger-sequencing-welcome-to-1996
subtitle: Today we’re going through a new method for identi…
tags:
- microbinfie
- podcast
- sars-cov-2
- genomic-surveillance
- variant-detection
- sanger-sequencing
- covid-19
- public-health
- bioinformatics
- diagnostic-methods
title: 'Episode 55: SARS-CoV-2 And Sequencing Spike With Sanger Sequencing - Welcome
  To 1995'
transcript: /microbinfie-transcripts/episode-55.txt
---

The microbinfie podcast explores an innovative method for rapid COVID-19 variant identification using Sanger sequencing, developed at the Technical University of Denmark to quickly detect and report variants of concern.

## Resources

- **Preprint**: [Link to preprint on medRxiv](https://www.medrxiv.org/content/10.1101/2021.03.27.21252266v1)
- **Protocol**: [Sanger sequencing of a part of the SARS-CoV-2 spike](https://www.protocols.io/view/sanger-sequencing-of-a-part-of-the-sars-cov-2-spik-bsbdnai6)
- **Software**: [GitHub repository for COVID spike classification software](https://github.com/kblin/covid-spike-classification)
- **Web App**: [SSI BioLib COVID Spike Classification App](https://ssi.biolib.com/app/covid-spike-classification/run)

**Note:** The method discussed is specific to the spike protein of _SARS-CoV-2_, and emphasizes the use of Sanger sequencing for variant identification.

### Key Points

#### 1. Rapid Variant Detection Method
- Developed a simplified Sanger sequencing approach for COVID-19 variant screening
- Uses existing RT-PCR infrastructure and ARTIC protocol primers
- Can process up to 200 samples per day with minimal additional resources

#### 2. Detection and Reporting Process
- Targets a 1,001-base region in the spike protein
- Results available within 50-60 hours from sample collection
- Immediate reporting to hospitals and contact tracing authorities

#### 3. Practical Implementation
- Initially focused on detecting B.1.1.7 variant
- Successfully identified first P1 variant in Denmark
- Serves as a preliminary screening before whole genome sequencing

### Take-Home Messages
- Rapid, accessible method for COVID-19 variant tracking
- Simplifies variant detection using existing laboratory infrastructure
- Enables quick public health response to emerging variants