---
audioLength: '22881063'
audioType: audio/mpeg
audioUrl: https://feeds.soundcloud.com/stream/1178439874-microbinfie-69-background-to-bacterial-taxonomy.mp3
author: Microbial Bioinformatics
date: '2021-12-16T05:00:30.000Z'
duration: 00:26:29
explicit: 'no'
guests:
- affiliation: Director, Australian Center for Ecogenomics, University of Queensland
  name: Phil Hugenholtz
  url: https://scmb.uq.edu.au/profile/324/phil-hugenholtz
- affiliation: Professor of Microbiology, Northumbria University; Chair, International
    Committee of Systematics and Prokaryotes
  name: Iain Sutcliffe
  url: https://www.northumbria.ac.uk/about-us/our-staff/s/iain-sutcliffe/
- affiliation: Professor of Microbial Genomics, University of East Anglia; Research
    Group Leader, Quadram Institute
  name: Mark Pallen
  url: https://www.cmi-norwich.ac.uk/people/prof-mark-pallen/
guid: tag:soundcloud,2010:tracks/1178439874
image: https://i1.sndcdn.com/artworks-hDzuZa9pRT5j9Lu2-hW4fUw-t3000x3000.png
link: https://soundcloud.com/microbinfie/69-background-to-bacterial-taxonomy
subtitle: There has been a lot of discussion about bacteria…
tags:
- species-definition
- bacterial-taxonomy
- microbial-nomenclature
- genomic-classification
- metagenomics
title: 'Episode 69: Background to bacterial taxonomy'
transcript: /microbinfie-transcripts/episode-69.txt
---

The microbinfie podcast explores the intricate world of bacterial taxonomy, tracing its historical development and philosophical foundations through conversations with leading experts in microbial systematics.

The discussion aims to clarify the fundamental aspects of bacterial taxonomy, a critical area for understanding microbial diversity, ecological roles, and evolutionary relationships. Changes in classification often stem from advancements in genetic and genomic analyses, leading to shifts in the understanding of bacterial phylogenetics. 
Bacterial taxonomy continues to evolve as scientific techniques advance, offering nuanced insights into microbial ecosystems. Engaging with experts in the field helps demystify complex changes and encourages informed discussions about the broader implications of taxonomic reclassifications.

Key Contributors:

- **Professor Phil Hugenholtz**: An expert in microbial ecology and evolution, contributing significant insights into microbial diversity and taxonomy.  
- **Professor Iain Sutcliffe**: Known for his work on bacterial systematics and the broader impact of taxonomy changes on related scientific fields.
- **Professor Mark Pallen**: Focuses on the genomics of pathogens and the implications of taxonomy in medical and environmental microbiology.

*Note: It's essential to remain updated with recent literature and to participate in discussions to stay informed about the ongoing developments in bacterial taxonomy.*


### Key Points

#### 1. Historical Development of Taxonomy
- Linnaeus established hierarchical classification and binomial nomenclature
- Darwin introduced evolutionary thinking and phylogenetic concepts
- Willi Hennig emphasized cladistic approaches over grade-based classifications

#### 2. Molecular Revolution in Taxonomy
- Carl Woese pioneered sequence-based bacterial classification
- Molecular techniques revealed fundamental differences between bacteria and archaea
- Challenged traditional prokaryotic classification paradigms

#### 3. Modern Taxonomic Challenges
- Ongoing debates about terminology like 'prokaryotes'
- International Committee for Systematic Prokaryotes (ICSP) maintains nomenclature standards
- Emphasis on phylogenetic accuracy over convenient terminology

### Take-Home Messages
- Taxonomy is a dynamic field driven by scientific understanding
- Molecular techniques have revolutionized biological classification
- Scientific precision matters in naming and categorizing organisms

## Selective Bibliography

For those interested in exploring the topics discussed in this episode further, here is a curated list of references (care of Mark Pallen) categorized by key figures and themes in bacterial taxonomy:

### Linnaeus and classification
- [10th edition of Systema Naturae](https://en.wikipedia.org/wiki/10th_edition_of_Systema_Naturae)
- W. T. Stearn. *The Background of Linnaeus's Contributions to the Nomenclature and Methods of Systematic Biology*. Systematic Zoology, 8 (1), Mar., 1959, pp. 4-22. [http://www.jstor.org/stable/2411603](http://www.jstor.org/stable/2411603)
- Daniel F. Austin. *The Nuance and Wit of Carolus Linnaeus*. The Palmetto 13(4), 1993. [PDF](https://www.fnps.org/assets/pdf/palmetto/v13i4p8austin.pdf)

### Darwin and evolution
- [Darwin Variorum Introduction](http://darwin-online.org.uk/Variorum/Introduction.html)
- Kevin Padian. *Charles Darwin's Views of Classification in Theory and Practice*. Syst. Biol. 48(2): 352-364, 1999. [https://doi.org/10.1080/106351599260337](https://doi.org/10.1080/106351599260337)

### Hennig and cladistics
- [Hennig 1966 - Phylogenetic Systematics (PDF)](http://aracnologia.macn.gov.ar/st/biblio/Hennig%201966%20Phylogenetic%20Systematics.pdf)
- Hennig, Willi (1975). *'Cladistic analysis or cladistic classification?': a reply to Ernst Mayr*. Systematic Zoology, 24 (2): 244-256. [https://doi.org/10.2307/2412765](https://doi.org/10.2307/2412765)
- Chi-Chun Ho, Susanna K. P. Lau, Patrick C. Y. Woo. *Romance of the three domains: how cladistics transformed the classification of cellular organisms*. Protein Cell 2013, 4(9): 664-676. [https://doi.org/10.1007/s13238-013-3050-9](https://doi.org/10.1007/s13238-013-3050-9)

### Woese, sequences and domains
- Woese CR. *Bacterial evolution*. Microbiol Rev. 1987 Jun;51(2): 221-71. [https://doi.org/10.1128/mr.51.2.221-271.1987](https://doi.org/10.1128/mr.51.2.221-271.1987)
- Pace NR. *Problems with "procaryote"*. J Bacteriol. 2009 Apr;191(7): 2008-10; discussion 2011. [https://doi.org/10.1128/JB.01224-08](https://doi.org/10.1128/JB.01224-08)

### Hugenholtz and GTDB
- Hugenholtz P, Pitulle C, Hershberger KL, Pace NR. *Novel division level bacterial diversity in a Yellowstone hot spring*. J Bacteriol. 1998 Jan;180(2): 366-76. [https://doi.org/10.1128/JB.180.2.366-376.1998](https://doi.org/10.1128/JB.180.2.366-376.1998)
- Parks DH, Chuvochina M, Waite DW, Rinke C, Skarshewski A, Chaumeil PA, Hugenholtz P. *A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life*. Nat Biotechnol. 2018 Nov;36(10): 996-1004. [https://doi.org/10.1038/nbt.4229](https://doi.org/10.1038/nbt.4229)
- Chaumeil PA, Mussig AJ, Hugenholtz P, Parks DH. *GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database*. Bioinformatics. 2019 Nov 15;36(6): 1925-7. [https://doi.org/10.1093/bioinformatics/btz848](https://doi.org/10.1093/bioinformatics/btz848)
- Parks DH, Chuvochina M, Chaumeil PA, Rinke C, Mussig AJ, Hugenholtz P. *A complete domain-to-species taxonomy for Bacteria and Archaea*. Nat Biotechnol. 2020 Sep;38(9): 1079-1086. [https://doi.org/10.1038/s41587-020-0501-8](https://doi.org/10.1038/s41587-020-0501-8)
- Parks DH, Chuvochina M, Rinke C, Mussig AJ, Chaumeil PA, Hugenholtz P. *GTDB: an ongoing census of bacterial and archaeal diversity through a phylogenetically consistent, rank normalized and complete genome-based taxonomy*. Nucleic Acids Res. 2021 Sep 14: gkab776. [https://doi.org/10.1093/nar/gkab776](https://doi.org/10.1093/nar/gkab776)
- Donovan H. Parks, Maria Chuvochina, Peter R. Reeves, Scott A. Beatson, Philip Hugenholtz. *Reclassification of Shigella species as later heterotypic synonyms of Escherichia coli in the Genome Taxonomy Database*. [https://doi.org/10.1101/2021.09.22.461432](https://doi.org/10.1101/2021.09.22.461432)

### Sutcliffe and modernizing bacterial taxonomy
- Sutcliffe IC. *Challenging the anthropocentric emphasis on phenotypic testing in prokaryotic species descriptions: rip it up and start again*. Front Genet. 2015 Jun 17;6: 218. [https://doi.org/10.3389/fgene.2015.00218](https://doi.org/10.3389/fgene.2015.00218)
- Renner SS. *A Return to Linnaeus's Focus on Diagnosis, Not Description: The Use of DNA Characters in the Formal Naming of Species*. Syst Biol. 2016 Nov;65(6): 1085-1095. [https://doi.org/10.1093/sysbio/syw032](https://doi.org/10.1093/sysbio/syw032)
- Murray AE, Freudenstein J, Gribaldo S, Hatzenpichler R, Hugenholtz P, Kämpfer P, Konstantinidis KT, Lane CE, Papke RT, Parks DH, Rossello-Mora R, Stott MB, Sutcliffe IC, Thrash JC, Venter SN, Whitman WB, et al. *Roadmap for naming uncultivated Archaea and Bacteria*. Nat Microbiol. 2020 Aug;5(8): 987-994. [https://doi.org/10.1038/s41564-020-0733-x](https://doi.org/10.1038/s41564-020-0733-x)
- Sutcliffe IC, Rosselló-Móra R, Trujillo ME. *Addressing the sublime scale of the microbial world: reconciling an appreciation of microbial diversity with the need to describe species*. New Microbes New Infect. 2021 Aug 13;43: 100931. [https://doi.org/10.1016/j.nmni.2021.100931](https://doi.org/10.1016/j.nmni.2021.100931)
- Vandamme P, Sutcliffe I. *Out with the old and in with the new: time to rethink twentieth century chemotaxonomic practices in bacterial taxonomy*. Int J Syst Evol Microbiol. 2021 Nov;71(11). [https://doi.org/10.1099/ijsem.0.005127](https://doi.org/10.1099/ijsem.0.005127)

### Pallen and modernizing bacterial nomenclature
- Gilroy R, Ravi A, Getino M, Pursley I, Horton DL, Alikhan NF, Baker D, Gharbi K, Hall N, Watson M, Adriaenssens EM, Foster-Nyarko E, Jarju S, Secka A, Antonio M, Oren A, Chaudhuri RR, La Ragione R, Hildebrand F, Pallen MJ. *Extensive microbial diversity within the chicken gut microbiome revealed by metagenomics and culture*. PeerJ. 2021 Apr 6;9: e10941. [https://10.7717/peerj.10941](https://10.7717/peerj.10941)
- Pallen MJ. *Bacterial nomenclature in the era of genomics*. New Microbe and New Infect 2021; 44: 100942. [https://doi.org/10.1016/j.nmni.2021.100942](https://doi.org/10.1016/j.nmni.2021.100942)
- Pallen MJ, Telatin A, Oren A. *The Next Million Names for Archaea and Bacteria*. Trends Microbiol. 2021 Apr;29(4): 289-298. [https://doi.org/10.1016/j.tim.2020.10.009](https://doi.org/10.1016/j.tim.2020.10.009)
- Pallen M, Alikhan NF. *Naming the Unnamed: Over 45,000 Candidatus Names for Unnamed Archaea and Bacteria in the Genome Taxonomy Database*. Preprints. [https://doi.org/10.20944/preprints202111.0557.v1](https://doi.org/10.20944/preprints202111.0557.v1)
