---
audioLength: '98393860'
audioType: audio/mpeg
audioUrl: https://feeds.soundcloud.com/stream/1179597649-microbinfie-bacterial-taxonomy-into-the-unknown-seqcode-naming-uncultured-multitudes-and-renaming-of-phyla.mp3
author: Microbial Bioinformatics
date: '2021-12-17T12:45:45.000Z'
duration: 00:40:59
explicit: 'no'
guests:
- affiliation: Director, Australian Center for Ecogenomics, University of Queensland
  name: Phil Hugenholtz
  url: https://scmb.uq.edu.au/profile/324/phil-hugenholtz
- affiliation: Professor of Microbiology, Northumbria University; Chair, International
    Committee of Systematics and Prokaryotes
  name: Iain Sutcliffe
  url: https://www.northumbria.ac.uk/about-us/our-staff/s/iain-sutcliffe/
- affiliation: Professor of Microbial Genomics, University of East Anglia; Research
    Group Leader, Quadram Institute
  name: Mark Pallen
  url: https://www.cmi-norwich.ac.uk/people/prof-mark-pallen/
guid: tag:soundcloud,2010:tracks/1179597649
image: https://i1.sndcdn.com/artworks-hDzuZa9pRT5j9Lu2-hW4fUw-t3000x3000.png
link: https://soundcloud.com/microbinfie/bacterial-taxonomy-into-the-unknown-seqcode-naming-uncultured-multitudes-and-renaming-of-phyla
subtitle: We finish our discussion on bacterial taxonomy, t…
tags:
- species-definition
- bacterial-taxonomy
- microbial-nomenclature
- genomic-classification
- metagenomics
title: 'Episode 71: Bacterial taxonomy: Into the unknown, SeqCode, naming uncultured
  multitudes and renaming of phyla'
transcript: /microbinfie-transcripts/episode-71.txt
---

The microbinfie podcast explores the intricate world of bacterial naming conventions and taxonomy, diving deep into the challenges of classifying and naming microbial species in the modern scientific landscape.

In our latest discussion on bacterial taxonomy, we explore the innovative methods for naming the multitude of unnamed, uncultured organisms as well as the contentious issue of renaming phyla. 

This discussion focuses on modern approaches in the taxonomy of bacteria, particularly on the challenges and methodologies involved in classifying unnamed and uncultured organisms. Additionally, we delve into the ongoing debate over the renaming of bacterial phyla, examining both the scientific and practical implications.

### Key Topics Discussed:

1. **Naming Uncultured Organisms**: 
   - Techniques and rationale behind naming bacteria that have not been cultured.

2. **Renaming of Phyla**: 
   - The reasons for and against the renaming of established phyla.
   - Insights into the controversies within the scientific community.

3. **Methodologies**: 
   - An exploration of the techniques and technologies used in the classification and naming of bacteria.
   - Discussion on the role of genomic data in contemporary bacterial taxonomy.

### Key Points

#### 1. Nomenclature and Latin Conventions
- Traditional bacterial naming relies on Latin and Greek roots
- Current nomenclature rules are flexible but contain specific recommendations
- Experts debate the importance of grammatical precision in scientific naming

#### 2. Challenges in Naming Uncultured Organisms
- Current systems struggle with naming species that cannot be cultured
- 'Candidatus' status provides a provisional naming mechanism
- Genomic data increasingly enables characterization of uncultured species

#### 3. Future of Bacterial Taxonomy
- Nomenclature codes are evolving and becoming more flexible
- Computational approaches enable generating large numbers of potential names
- Priority and name stability remain critical considerations

### Take-Home Messages
- Bacterial naming is a complex, evolving scientific practice
- Flexibility in nomenclature is increasingly important
- Genomic approaches are transforming how we classify microbial species

## Selective Bibliography

For those interested in exploring the topics discussed in this episode further, here is a curated list of references (care of Mark Pallen) categorized by key figures and themes in bacterial taxonomy:

### Linnaeus and classification
- [10th edition of Systema Naturae](https://en.wikipedia.org/wiki/10th_edition_of_Systema_Naturae)
- W. T. Stearn. *The Background of Linnaeus's Contributions to the Nomenclature and Methods of Systematic Biology*. Systematic Zoology, 8 (1), Mar., 1959, pp. 4-22. [http://www.jstor.org/stable/2411603](http://www.jstor.org/stable/2411603)
- Daniel F. Austin. *The Nuance and Wit of Carolus Linnaeus*. The Palmetto 13(4), 1993. [PDF](https://www.fnps.org/assets/pdf/palmetto/v13i4p8austin.pdf)

### Darwin and evolution
- [Darwin Variorum Introduction](http://darwin-online.org.uk/Variorum/Introduction.html)
- Kevin Padian. *Charles Darwin's Views of Classification in Theory and Practice*. Syst. Biol. 48(2): 352-364, 1999. [https://doi.org/10.1080/106351599260337](https://doi.org/10.1080/106351599260337)

### Hennig and cladistics
- [Hennig 1966 - Phylogenetic Systematics (PDF)](http://aracnologia.macn.gov.ar/st/biblio/Hennig%201966%20Phylogenetic%20Systematics.pdf)
- Hennig, Willi (1975). *'Cladistic analysis or cladistic classification?': a reply to Ernst Mayr*. Systematic Zoology, 24 (2): 244-256. [https://doi.org/10.2307/2412765](https://doi.org/10.2307/2412765)
- Chi-Chun Ho, Susanna K. P. Lau, Patrick C. Y. Woo. *Romance of the three domains: how cladistics transformed the classification of cellular organisms*. Protein Cell 2013, 4(9): 664-676. [https://doi.org/10.1007/s13238-013-3050-9](https://doi.org/10.1007/s13238-013-3050-9)

### Woese, sequences and domains
- Woese CR. *Bacterial evolution*. Microbiol Rev. 1987 Jun;51(2): 221-71. [https://doi.org/10.1128/mr.51.2.221-271.1987](https://doi.org/10.1128/mr.51.2.221-271.1987)
- Pace NR. *Problems with "procaryote"*. J Bacteriol. 2009 Apr;191(7): 2008-10; discussion 2011. [https://doi.org/10.1128/JB.01224-08](https://doi.org/10.1128/JB.01224-08)

### Hugenholtz and GTDB
- Hugenholtz P, Pitulle C, Hershberger KL, Pace NR. *Novel division level bacterial diversity in a Yellowstone hot spring*. J Bacteriol. 1998 Jan;180(2): 366-76. [https://doi.org/10.1128/JB.180.2.366-376.1998](https://doi.org/10.1128/JB.180.2.366-376.1998)
- Parks DH, Chuvochina M, Waite DW, Rinke C, Skarshewski A, Chaumeil PA, Hugenholtz P. *A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life*. Nat Biotechnol. 2018 Nov;36(10): 996-1004. [https://doi.org/10.1038/nbt.4229](https://doi.org/10.1038/nbt.4229)
- Chaumeil PA, Mussig AJ, Hugenholtz P, Parks DH. *GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database*. Bioinformatics. 2019 Nov 15;36(6): 1925-7. [https://doi.org/10.1093/bioinformatics/btz848](https://doi.org/10.1093/bioinformatics/btz848)
- Parks DH, Chuvochina M, Chaumeil PA, Rinke C, Mussig AJ, Hugenholtz P. *A complete domain-to-species taxonomy for Bacteria and Archaea*. Nat Biotechnol. 2020 Sep;38(9): 1079-1086. [https://doi.org/10.1038/s41587-020-0501-8](https://doi.org/10.1038/s41587-020-0501-8)
- Parks DH, Chuvochina M, Rinke C, Mussig AJ, Chaumeil PA, Hugenholtz P. *GTDB: an ongoing census of bacterial and archaeal diversity through a phylogenetically consistent, rank normalized and complete genome-based taxonomy*. Nucleic Acids Res. 2021 Sep 14: gkab776. [https://doi.org/10.1093/nar/gkab776](https://doi.org/10.1093/nar/gkab776)
- Donovan H. Parks, Maria Chuvochina, Peter R. Reeves, Scott A. Beatson, Philip Hugenholtz. *Reclassification of Shigella species as later heterotypic synonyms of Escherichia coli in the Genome Taxonomy Database*. [https://doi.org/10.1101/2021.09.22.461432](https://doi.org/10.1101/2021.09.22.461432)

### Sutcliffe and modernizing bacterial taxonomy
- Sutcliffe IC. *Challenging the anthropocentric emphasis on phenotypic testing in prokaryotic species descriptions: rip it up and start again*. Front Genet. 2015 Jun 17;6: 218. [https://doi.org/10.3389/fgene.2015.00218](https://doi.org/10.3389/fgene.2015.00218)
- Renner SS. *A Return to Linnaeus's Focus on Diagnosis, Not Description: The Use of DNA Characters in the Formal Naming of Species*. Syst Biol. 2016 Nov;65(6): 1085-1095. [https://doi.org/10.1093/sysbio/syw032](https://doi.org/10.1093/sysbio/syw032)
- Murray AE, Freudenstein J, Gribaldo S, Hatzenpichler R, Hugenholtz P, Kämpfer P, Konstantinidis KT, Lane CE, Papke RT, Parks DH, Rossello-Mora R, Stott MB, Sutcliffe IC, Thrash JC, Venter SN, Whitman WB, et al. *Roadmap for naming uncultivated Archaea and Bacteria*. Nat Microbiol. 2020 Aug;5(8): 987-994. [https://doi.org/10.1038/s41564-020-0733-x](https://doi.org/10.1038/s41564-020-0733-x)
- Sutcliffe IC, Rosselló-Móra R, Trujillo ME. *Addressing the sublime scale of the microbial world: reconciling an appreciation of microbial diversity with the need to describe species*. New Microbes New Infect. 2021 Aug 13;43: 100931. [https://doi.org/10.1016/j.nmni.2021.100931](https://doi.org/10.1016/j.nmni.2021.100931)
- Vandamme P, Sutcliffe I. *Out with the old and in with the new: time to rethink twentieth century chemotaxonomic practices in bacterial taxonomy*. Int J Syst Evol Microbiol. 2021 Nov;71(11). [https://doi.org/10.1099/ijsem.0.005127](https://doi.org/10.1099/ijsem.0.005127)

### Pallen and modernizing bacterial nomenclature
- Gilroy R, Ravi A, Getino M, Pursley I, Horton DL, Alikhan NF, Baker D, Gharbi K, Hall N, Watson M, Adriaenssens EM, Foster-Nyarko E, Jarju S, Secka A, Antonio M, Oren A, Chaudhuri RR, La Ragione R, Hildebrand F, Pallen MJ. *Extensive microbial diversity within the chicken gut microbiome revealed by metagenomics and culture*. PeerJ. 2021 Apr 6;9: e10941. [https://10.7717/peerj.10941](https://10.7717/peerj.10941)
- Pallen MJ. *Bacterial nomenclature in the era of genomics*. New Microbe and New Infect 2021; 44: 100942. [https://doi.org/10.1016/j.nmni.2021.100942](https://doi.org/10.1016/j.nmni.2021.100942)
- Pallen MJ, Telatin A, Oren A. *The Next Million Names for Archaea and Bacteria*. Trends Microbiol. 2021 Apr;29(4): 289-298. [https://doi.org/10.1016/j.tim.2020.10.009](https://doi.org/10.1016/j.tim.2020.10.009)
- Pallen M, Alikhan NF. *Naming the Unnamed: Over 45,000 Candidatus Names for Unnamed Archaea and Bacteria in the Genome Taxonomy Database*. Preprints. [https://doi.org/10.20944/preprints202111.0557.v1](https://doi.org/10.20944/preprints202111.0557.v1)
