---
date: 2022-05-26 00:00:00
title: MicroBinfie Podcast, 83 A short journey into mobile genetic elements
link: https://soundcloud.com/microbinfie/a-short-journey-into-mobile-genetic-elements
tags:
  - microbinfie
  - podcast
---
# Discussion on Mobile Genetic Elements in Bacteria

In our recent discussion, we delved into the fascinating yet complex world of mobile genetic elements in bacteria. This topic is particularly challenging due to the intricate nature of genetic transfer and variability in microbial life.

During our conversation, we had a brief but informative chat. However, we encountered a hiccup as Lee unfortunately lost power. Despite this interruption, rest assured that we plan to return with a more comprehensive Part 2 in the near future.

Stay tuned for more insights and an in-depth exploration in our upcoming session!

### Extra notes

- **Mobile Genetic Elements (MGEs):** Discussed various MGEs including bacteriophages, transposons, plasmids, genomic islands, integrative conjugative elements, and integrons. These elements can move between genomes and chromosomes, complicating genomic analysis due to their diverse nature and mechanisms.

- **Challenges with Genome Assembly:** Short-read sequencing can result in assembly issues, especially in repeated or repetitive genome regions. The discussion highlighted issues with Vibrio strains and pertussis due to high repeat content, which are challenging for short-read assemblies and might require long-read technologies like PacBio.

- **CRISPR Systems:** Examined CRISPR as a bacterial immune mechanism against phages, noting its potential for genomic surveillance, although its effectiveness varies among organisms. Multiple CRISPR detection tools were mentioned, including CRISPR Finder, PileCR, and others.

- **Annotation Tools and Strategies:** Initial genome annotation can be done with tools like Prokka, followed by more specific annotators for elements like IS elements or plasmids. Annotation helps identify MGEs by synteny of genes.

- **Identifying MGEs:** Methods to identify MGEs include mapping sequencing reads back to assemblies to check for coverage anomalies, using GC content differences, and utilizing tools like Plasmid Finder, PhageFind, PhySpy, and others.

- **Use of Genomic Browsers:** Visualization tools like Artemis are crucial for manual inspection of genome annotations to accurately identify and understand MGEs, as reliance on algorithms alone may not be sufficient.

- **Technical Limitations:** Discussed the limitations of current bioinformatics tools, stressing the need for manual inspection and collaboration with laboratory work to fully understand uncharacterized genomic content.

- **Public Databases and Tool Efficiency:** Despite numerous databases and tools available for phage and plasmid detection, they may not always provide conclusive results, highlighting the complexity of microbial genomics.

- **Leveraging Community and Databases:** When faced with unknown annotations, discussing findings with peers and using curated databases like ProPhage or plasma databases can provide additional insights, although they might not always be helpful.

These insights emphasize the challenges and importance of carefully selecting and using computational tools in microbial bioinformatics, highlighting the need for integrating bioinformatics with laboratory research.

The speakers on the MicroBinfeed podcast are Dr. Nabil Ali Khan, Dr. Andrew Page, and Dr. Lee Katz.

