---
title: Nabil-Fareed Alikhan - Curriculum Vitae
slug: "/cv"
tags:
  - page
---

I am a bioinformatician with 11 years experience. 

# Employment

**Bioinformatics Scientific Programmer Dec 2018 - Current**

Quadram Institute Bioscience, Norwich, United Kingdom

Duties include:

- COG UK Consortium National effort for tracking COVID19 (SARSCoV2) through genome sequencing - I contribute analysis and data curation.
- Building better bioinformatics infrastructure, moving towards open web platforms and cloud computing - I co-develop and maintain our in-house platforms in the institute.
- Population genetics and pathogenesis of enteric pathogens, including _Salmonella_ and _E. coli_
- The Public Health Alliance for Genomic Epidemiology (PHA4GE) - infrastructure working group, aiming to find consensus in how we deploy Bioinformatics for public health across the world.
- Funded by Quadram Institute Bioscience BBSRC funded Core Capability Grant until March 2028.

**Senior Research Fellow in Pathogen Bioinformatics Sep 2016 – Dec 2018**

Warwick Medical School, University of Warwick, United Kingdom

Duties included integrating genomes from ancient metagenomes into EnteroBase, and the sequencing and analysis of 10,000 modern _S. enterica_. On-going development of EnteroBase and other software.

**Research Fellow in Genomics of Bacterial Pathogens Dec 2014 – Sep 2016**

Warwick Medical School, University of Warwick, United Kingdom

Duties included co-development of EnteroBase (enterobase.warwick.ac.uk), a web-resource for &#39;big data&#39; genomic analysis of Enteropathogens and adapting existing robotic genotyping to perform library preparations for high-throughput sequencing.

**Research Associate Mar 2009 – Nov 2009**

School of Chemistry and Molecular Biosciences, University of Queensland, Australia

Duties included system administration and technical support for Linux servers and MacOSX workstations within the Beatson Microbial Genomics group.

**Research Associate Nov 2008 – Jan 2009**

School of Chemistry and Molecular Biosciences, University of Queensland, Australia

Duties included programming software pipelines for bioinformatics especially for parallelization on high performance computer clusters; and server administration for sharing bacterial genomics data.

# Education 

**Doctor of Philosophy (Microbiology) 2010 – 2015**

University of Queensland, Australia

Primary Supervisor: Scott Beatson

Thesis title: Escherichia coli virulence: a genomic approach

**Bachelor of Science (Honours – 1st Class) (Microbiology) 2009**

University of Queensland, Australia

Thesis title: Comparative genome analysis of Escherichia coli VR 50

**Bachelor of Science (Biochemistry) &amp; Bachelor of Information Technology 2004 - 2008**

University of Queensland, Australia

# Skills

**Bioinformatics**

- De novo assembly and mapping of sequencing reads
- Comparative genome analysis
- Phylogenetic inference e.g. PhyML &amp; RAxML
- Statistics: Python&#39;s matplotlib/numpy/scipy libraries and R.
- Linux &amp; command-line environment

**Software Development**

- Development of EnteroBase, a web-resource serving over 600,000 genome assemblies, providing genotyping, genome analysis and curated metadata for over 1,600 users in a range of major institutions.
- Development of BRIG software for visualisation of whole genome comparisons.
- Developing novel visualisation and large-scale analysis tools, e.g. GrapeTree
- Pipelines for high performance parallel computing architectures

**Programming Languages**

- Full Stack Developer: Python, Java, Bash &amp; Perl, Relationship databases (SQL), Web-frameworks (Python-Flask), JavaScript
